src/Lightning.ts

```diff
@@ -271,7 +272,7 @@ export const LightningMixin = (superclass) =>
         features,
         max_fee,
       } = await validate({ params, logger: lightningLogger })
-      const { memo: memoPayer } = params
+      const { memo: memoPayer, twoFactorToken: token } = params
```

Comment:
why the renaming? we could keep `twoFactorToken`?

Reply:
we have `token` and `tokens`

Reply:
The `node-2fa` library uses the `token` variable name, so to keep naming consistent in functions that use functions from the library, I am renaming `twoFactorToken` to `token`.
But I do see your point that having both `token` and `tokens` in the same function can be confusing. I'll refactor the name